lib/src/ktlint/java/com/diffplug/spotless/glue/ktlint/KtlintFormatterFunc.java

```diff
@@ -16,19 +16,14 @@
 package com.diffplug.spotless.glue.ktlint;
 
 import java.io.File;
+import java.nio.file.Path;
 import java.util.Map;
 
 import org.jetbrains.annotations.NotNull;
 
+import com.diffplug.spotless.FileSignature;
 import com.diffplug.spotless.FormatterFunc;
-import com.diffplug.spotless.glue.ktlint.compat.KtLintCompat0Dot31Dot0Adapter;
-import com.diffplug.spotless.glue.ktlint.compat.KtLintCompat0Dot32Dot0Adapter;
-import com.diffplug.spotless.glue.ktlint.compat.KtLintCompat0Dot34Dot2Adapter;
-import com.diffplug.spotless.glue.ktlint.compat.KtLintCompat0Dot45Dot2Adapter;
-import com.diffplug.spotless.glue.ktlint.compat.KtLintCompat0Dot46Dot0Adapter;
-import com.diffplug.spotless.glue.ktlint.compat.KtLintCompat0Dot47Dot0Adapter;
-import com.diffplug.spotless.glue.ktlint.compat.KtLintCompat0Dot48Dot0Adapter;
-import com.diffplug.spotless.glue.ktlint.compat.KtLintCompatAdapter;
+import com.diffplug.spotless.glue.ktlint.compat.*;
```

Comment:
@nedtwigg What's the project style guide when it comes to star imports?

Reply:
I don't see that IntelliJ Idea CE 2022.3.1 is using settings from the `.editorconfig` file, even I press "apply from EditorConfig" button. 

Also `./gradlew spotlessApply` doesn't change this as well and `./gradlew spotlessCheck` passes. If this is crucial for the project, I'd like to have these settings used at least in `spotlessCheck/Apply` tasks. (even this will require manual settings).

Additionally, command`git grep 'import .*\*.*'` shows quite a lot of files formatted this way.

Reply:
You're right: if it's not enforced automatically, you shouldn't have to worry about it.